terraform/digitalocean/provider.tf

```diff
@@ -2,3 +2,11 @@ provider "digitalocean" {
   token = var.do_token
   version = "~> 1.16"
 }
+
+terraform {
```

Comment:
This is required. Terraform won't build `digitalocean` without it.

Reply:
True! Thanks for the fix!

Reply:
np!